index_test.go

```diff
@@ -128,8 +128,6 @@ func checkFatal(t *testing.T, err error) {
 	// The failure happens at wherever we were called, not here
 	_, file, line, ok := runtime.Caller(1)
 	if !ok {
-		t.Fatal()
+		t.Fatalf("Fail at %v:%v; %v", file, line, err)
 	}
-
-	t.Fatalf("Fail at %v:%v; %v", file, line, err)
```

Comment:
This is correct, the `!ok` block refers to the `runtime.Caller(1)` call, it says nothing about the test.